open_event/static/js/jquery/jquery.smartWizard.js

```diff
@@ -0,0 +1,489 @@
+            $this.target
+        );
+    };
+
+    var _loadContent = function($this, stepIdx) {
+        var selStep = $this.steps.eq(stepIdx);
+        var ajaxurl = $this.options.contentURL;
+        var ajaxurl_data = $this.options.contentURLData;
+        var hasContent = selStep.data('hasContent');
+        var stepNum = stepIdx+1;
+        if (ajaxurl && ajaxurl.length>0) {
+            if ($this.options.contentCache && hasContent) {
+                _showStep($this, stepIdx);
+            } else {
+                var ajax_args = {
+                    url: ajaxurl,
+                    type: $this.options.ajaxType,
+                    data: ({step_number : stepNum}),
+                    dataType: "text",
+                    beforeSend: function(){
+                        $this.loader.show();
+                    },
+                    error: function(){
+                        $this.loader.hide();
+                    },
+                    success: function(res){
+                        $this.loader.hide();
+                        if(res && res.length>0){
+                            selStep.data('hasContent',true);
+                            _step($this, selStep).html(res);
+                            _showStep($this, stepIdx);
+                        }
+                    }
+                };
+                if (ajaxurl_data) {
+                    ajax_args = $.extend(ajax_args, ajaxurl_data(stepNum));
+                }
+                $.ajax(ajax_args);
+            }
+        }else{
+            _showStep($this,stepIdx);
+        }
+    };
+
+    var _showStep = function($this, stepIdx) {
+        var selStep = $this.steps.eq(stepIdx);
+        var curStep = $this.steps.eq($this.curStepIdx);
+        if(stepIdx != $this.curStepIdx){
+            if($.isFunction($this.options.onLeaveStep)) {
+                var context = { fromStep: $this.curStepIdx+1, toStep: stepIdx+1 };
+                if (! $this.options.onLeaveStep.call($this,$(curStep), context)){
+                    return false;
+                }
+            }
+        }
+        //$this.elmStepContainer.height(_step($this, selStep).outerHeight());
+        var prevCurStepIdx = $this.curStepIdx;
+        $this.curStepIdx =  stepIdx;
+        if ($this.options.transitionEffect == 'slide'){
+            _step($this, curStep).slideUp("fast",function(e){
+                _step($this, selStep).slideDown("fast");
+                _setupStep($this,curStep,selStep);
+            });
+        } else if ($this.options.transitionEffect == 'fade'){
+            _step($this, curStep).fadeOut("fast",function(e){
+                _step($this, selStep).fadeIn("fast");
+                _setupStep($this,curStep,selStep);
+            });
+        } else if ($this.options.transitionEffect == 'slideleft'){
+            var nextElmLeft = 0;
+            var nextElmLeft1 = null;
+            var nextElmLeft = null;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Variable 'nextElmLeft' is already declared in line 226.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2677245224/issues/source?bid=3233864&fileBranchId=3324174#l228)